tests/test_main_startup_rollback.py

```
@@ -0,0 +1,403 @@
+
+
+@patch("mmrelay.main.initialize_database")
+@patch("mmrelay.main.load_plugins")
+@patch("mmrelay.main.start_message_queue")
+@patch("mmrelay.main.connect_matrix")
+@patch("mmrelay.main.connect_meshtastic")
+@patch("mmrelay.main._remove_ready_file")
+@patch("mmrelay.main.shutdown_plugins")
+@patch("mmrelay.main.stop_message_queue")
+def test_startup_rollback_cancels_check_connection_task(
+    _mock_stop_queue,
```

Ruff (0.15.12)

[warning] Fixture `_mock_stop_queue` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)